src/ansys/turbogrid/core/multi_blade_row/multi_blade_row.py

```diff
@@ -47,6 +47,29 @@
 import ansys.turbogrid.core.ndf_parser.ndf_parser as ndf_parser
 
 
+class MachineSizingStrategy(IntEnum):
+    """
+    These are machine sizing strategies that can be optionally applied using set_machine_sizing_strategy.
+
+    Description
+    -----------
+    NONE
+        No strategy applied. Blade Row sizings must be manually controlled via either of:
```

Comment:
@Acohen-work I would replace "either" with "any" (at least in my British English, we tend to use either when there are only two choices).